### PR TITLE
grants page design and copy cleanup

### PR DESCRIPTION
**Pull request by gordoncc**
took the liberty to tidy up this page for clarity and legibility

### CODE REVIEW

File: pages/grants.js [hunk 1]
```diff
@@ -129,158 +129,140 @@ export default function Grants({ posts, categories, types, search }) {
...
+                  <Link href="/grants/proposals" passHref>
+                    <a className="button-sm bg-blue-400 text-white mb-8 max-w-fit">
+                      Submit a Proposal
+                    </a>
+                  </Link>
+
+                  {/* Apprenticeships */}
+                    <h3>Apprenticeships</h3>
+                    <p className="mb-4">
+                     Practical learning opportunities for
```
Comment (mikeosborne) [inline pages/grants.js]:
```suggestion
                     Apprenticeships are practical learning opportunities for
```
Sentence fragment

File: pages/grants.js [hunk 2]
```diff
@@ -129,158 +129,140 @@ export default function Grants({ posts, categories, types, search }) {
...
+                    new Urbit developers provided by members of the Urbit community.
+                    They're one of the best ways to level up your skills, and often
```
Comment (mikeosborne) [inline pages/grants.js]:
```suggestion
                    They're one of the best ways to level up your skills and often
```

File: pages/grants.js [hunk 3]
```diff
@@ -129,158 +129,140 @@ export default function Grants({ posts, categories, types, search }) {
...
-                  <a className="button-sm bg-wall-400 text-white max-w-fit">
-                    Learn more
-                  </a>
-                </Link>
-              </div>
+                <div className="md:basis-1/2">
+                  {/* Proposals */}
+                    <h3>Proposals</h3>
+                    <p className="mb-4">
+                    Get funding for your project.  We fund all kinds projects, not strictly technical ones,
```
Comment (mikeosborne) [inline pages/grants.js]:
```suggestion
                    Proposals are the main way to get funding for your project.  We fund all kinds projects, not strictly technical ones,
```
Sentence fragment

File: pages/grants.js [hunk 4]
```diff
@@ -129,158 +129,140 @@ export default function Grants({ posts, categories, types, search }) {
...
+                </p>
+
+                <JoinGroup
+                  emphasize
+                  className="mb-8"
+                  groupName="~wolref-podlex/foundation"
+                />
+
+                <p>
+                  If you're not on the network, reach out to us at{" "}
```
Comment (mikeosborne) [inline pages/grants.js]: Shouldn't we point them to the Getting Started page instead?

Comment (gordoncc) [reply pages/grants.js]: Yeah, if you're getting grant money you should probably be on the network.  Good point.

File: pages/grants.js [hunk 5]
```diff
@@ -129,158 +129,140 @@ export default function Grants({ posts, categories, types, search }) {
...
+                    </a>
+                  </Link>
+                </div>
+
+                <div className="md:basis-1/2">
+                  {/* Bounties */}
+                  <h3>Bounties</h3>
+                  <p className="mb-4">
+                  Contracts for work provided by either the
+                    Urbit Foundation or from trusted partners in our ecosystem.
```
Comment (mikeosborne) [inline pages/grants.js:182]:
```suggestion
                    Urbit Foundation or from trusted partners in the Urbit universe.
```
I thought we weren't using the word "ecosystem"

File: pages/grants.js [hunk 6]
```diff
@@ -129,158 +129,140 @@ export default function Grants({ posts, categories, types, search }) {
...
+                      Become an Apprentice
+                    </a>
+                  </Link>
+                </div>
+
+                <div className="md:basis-1/2">
+                  {/* Bounties */}
+                  <h3>Bounties</h3>
+                  <p className="mb-4">
+                  Contracts for work provided by either the
```
Comment (mikeosborne) [inline pages/grants.js]:
```suggestion
                  Bounties are contracts for work provided by either the
```
Sentence fragment

File: pages/grants.js (hunk 5 above)
Comment (gordoncc) [reply pages/grants.js:182]: This is one of those things like Mars that's contextual.  People know what a software ecosystem is, a software universe is an unfamiliar term.  So I'd like to say "ecosystem" on public facing pages and on internal communications we can use all our in-house terms.

Screen Shot 2022-07-18 at 11 44 18 AM

**Review by jalehman — APPROVED** [review]

**Pull request merged**